### PR TITLE
Convert next steps to stop using Eligibility Paths

### LINKED ISSUE #1692: Convert other AirTable tables to new benefitEligibility format

### PR DESCRIPTION
**Pull request by brdunfield**
Closes #1692 and #1693

### CODE REVIEW

File: selectors/benefits.js [hunk 1]
```diff
@@ -183,40 +181,35 @@ export const getFilteredBenefits = createSelector(
...
       if (profileFilters[criteria] !== "") hasSelections = true;
     });
 
-    if (hasSelections)
+    if (hasSelections) {
+      // only check eligible nextSteps if we know we have selections
+      let eligibleNextStepIds = [];
+      nextSteps.forEach(ns => {
+        if (
+          benefitEligibilityMatch(ns, profileFilters, multipleChoiceOptions)
```
Comment (smcmurtry) [inline selectors/benefits.js:197]: maybe we should rename this function to be `eligibilityMatch` if it's going to be used for next steps too.

**Review by smcmurtry — APPROVED** [review]
Looks good, one minor comment. I think we should eventually make a separate nextStepsEligibility sheet so we can have multiple rules for a single next step.

File: selectors/benefits.js (hunk 1 above)
Comment (brdunfield) [reply selectors/benefits.js:197]: I think that this could be done as part of an upcoming PR to remove eligibilityPaths from the code.

**Comment by brdunfield** [comment]
[Change eligibility path structure to be more like Allison's spreadsheet](https://trello.com/c/KXov9hiW/148-change-eligibility-path-structure-to-be-more-like-allisons-spreadsheet)

**Pull request merged**